connector/src/main/java/it/eng/connector/model/User.java

```diff
@@ -1,20 +1,17 @@
 package it.eng.connector.model;
 
-import java.io.Serial;
-import java.util.Collection;
-import java.util.List;
-
-import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
```

Comment:
In fact, this is the code that you wrote, so I left it as is.
When I tried to delete the @AllArgsConstructor, @Builder is not feasible, since Lombok @Builder cannot automatically assume how to handle the object's construction without  an appropriate constructor, so I assume that is the reason why you left it there. So it's either using both of them, or write custom Builder.

And it's not present in other model classes, since in other we implemented the custom builder, and we're not using Lombok's one.